rel/kazoo

```diff
@@ -40,6 +40,7 @@ ERL_OPTS="{{ erl_opts }}"
 RUNNER_LOG_DIR="${RUNNER_LOG_DIR:-/tmp/log}"
 VM_NODE_FILES_DIR="${VM_NODE_FILES_DIR:-/tmp}"
 NAME_TYPE="${NODE_NAME_TYPE:--name}"
+NODE_STARTUP_FILE_PATH=${NODE_STARTUP_FILE_PATH:-/etc/kazoo/core}
```

Comment:
@lazedo as per the standup, this looks fine but makes me nerves and we agreed I owe you a shot as well as a beer (at least) if someone needs this functionality.  ;)

Reply:
i pushed GC change in different PRs. the 4.3 of this PR should be merged after next release.